Jenkinsfile

```diff
@@ -53,3 +53,23 @@ for (int i = 0; i < platforms.size(); ++i) {
 
 /* Execute our platforms in parallel */
 parallel(branches)
+
+node('docker') {
+    checkout scm
+
+    stage('Build Docker Image') {
+        sh 'make docker'
+    }
+
+    stage("Run known successful case(s)") {
```

Comment:
TODO: prepare already more a `parallel` data structure, to run many variations in parallel, so it's easy to add more as we go: a must-fail one, etc.

Reply:
TODO 2:
* download a Jenkins WAR and map it, so the testing stability is not subject to the new available weeklies